src/app/editor-toolbar-container/editor-toolbar-save/editor-toolbar-save.component.ts

```diff
@@ -1,8 +1,13 @@
-import { Component } from '@angular/core';
+import { Component, Input, Output, EventEmitter } from '@angular/core';
+
 
 @Component({
   selector: 'me-editor-toolbar-save',
   templateUrl: './editor-toolbar-save.component.html',
   styleUrls: ['../../multi-editor/multi-editor.component.scss']
 })
-export class EditorToolbarSaveComponent { }
+export class EditorToolbarSaveComponent {
```

Comment:
I am not sure if this supposed to be a component.

Reply:
@zzacharo  ditto

Reply:
This is gonna have some save logic in the future.